pentaho-platform-plugin-deployer/src/main/java/org/pentaho/osgi/platform/plugin/deployer/impl/PluginZipFileProcessor.java

```diff
@@ -328,6 +338,42 @@ public boolean accept( File f ) {
     }
   }
 
+  public void processManifest( ZipInputStream zipInputStream, ZipOutputStream zipOutputStream ) throws Exception {
```

Comment:
This code is very similar to the code in lines 275-288. Can't we also call the function there?